test-packages/basic-app/test/asset-rewriting-test.js

```diff
@@ -0,0 +1,21 @@
+const chai = require('chai');
+const expect = chai.expect;
+const glob = require("glob");
+const execa = require("execa");
+
+chai.use(require('chai-fs'));
+
+describe.only('rewriting HTML', function () {
```

Comment:
```suggestion
describe('rewriting HTML', function () {
```
😉 

I wonder if we can enable a lint rule to catch these?

Reply:
😞  thanks heaps @mansona